hal/randombytes.c

```diff
@@ -1,6 +1,15 @@
 // SPDX-License-Identifier: Apache-2.0
 #include "randombytes.h"
 
+#ifdef RISCV
+#include <bootloader_random.h>
+int randombytes(uint8_t *obuf, size_t len) {
+    bootloader_fill_random(obuf, len);
```

Comment:
Can you put a comment here about what's actually underlying `bootloader_fill_random`? 
 A statement regarding how secure the RNG is would be good.  Maybe a reference to the manual?

Reply:
I've added some comment for this, please have a look